src/algebra/order/hom/ring.lean

```diff
@@ -294,5 +299,54 @@ def to_order_ring_hom (f : α ≃+*o β) : α →+*o β :=
 @[simp]
 lemma coe_to_order_ring_hom_refl : (order_ring_iso.refl α : α →+*o α) = order_ring_hom.id α := rfl
 
+lemma to_order_ring_hom_injective : injective (to_order_ring_hom : (α ≃+*o β) → α →+*o β) :=
+λ f g h, fun_like.coe_injective $ by convert fun_like.ext'_iff.1 h
+
 end non_assoc_semiring
 end order_ring_iso
+
+/-!
+### Unicity
```

Comment:
"Uniqueness"?

Reply:
Do you prefer that? [Wiktionary says they are synonyms](https://en.wiktionary.org/wiki/unicity).

Reply:
Nevertheless, https://english.stackexchange.com/a/225098/2186 says it rather well. :-)

Reply:
Oh well
```suggestion
### Uniqueness
```